packages/block-editor/src/hooks/duotone.js

```diff
@@ -251,10 +251,6 @@ function useDuotoneStyles( {
 	const selectors = duotoneSelector.split( ',' );
 
 	const selectorsScoped = selectors.map( ( selectorPart ) => {
-		// Extra .editor-styles-wrapper specificity is needed in the editor
-		// since we're not using inline styles to apply the filter. We need to
-		// override duotone applied by global styles and theme.json.
-
```

Comment:
The selector no longer has `.editor-styles-wrapper` added, though it looks like the comment wasn't removed.